packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -170,6 +170,41 @@ export const diffusers = (model: ModelData): string[] => {
 	}
 };
 
+export const diffusionkit = (): string[] => {
+	const sd3Snippet = `from diffusionkit.mlx import DiffusionPipeline
+
+pipeline = DiffusionPipeline(
+	shift=3.0,
+	use_t5=False,
+	model_version="argmaxinc/mlx-stable-diffusion-3-medium",
```

Comment:
So is it not possible to do it model-specific at this time, as @Wauplin mentioned?

Reply:
Agree that it would be best to showcase only the relevant code snippet here. How does the CLI determines which pipeline to instantiate? I would reuse the same logic here. Otherwise, it's also possible to tag the models on the Hub as `"flux"` or `stable-diffusion` (using the `tags` list in the model card metadata) and use this information to generate the snippets.

Reply:
Thank you for your feedback! Both of your proposals are excellent suggestions. I will be implementing the latter approach, using tags in the model card metadata to determine which snippets to generate.